### PR TITLE
Mirror Chrome -> WebView for css/*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR mirrors the values of Chrome Desktop to WebView Android is `true` or `null`, to help reduce data inconsistencies and reach towards the 2019 Key Result.  Notes have been adapted to ensure they are not mentioning Chrome versions earlier than 37.

The script that performed the majority of changes can be found here: https://github.com/vinyldarkscratch/bcd-toolkit/blob/master/fix.py

### CODE REVIEW

**Comment by ddbeck** [comment]
I keep thinking we might want a generic and tested mirroring script so simple cases can be done through a perfunctory bulk process and then we could break out the special cases (notes, partials, etc.) that could get a real review. This seems like a likely candidate for that sort thing.

(Which reminds me I need to review your other bulk update scripts. Sorry for the delay!)

**Comment by vinyldarkscratch** [comment]
I'm on board for that idea, yeah!

**Comment by vinyldarkscratch** [comment]
I've just posted a link to the mirroring script I wrote (It's in Python, but I promise it doesn't bite!), which takes a browser as an argument and copies values from the applicable parent browser.  I think we could easily make it a runnable JavaScript script called "mirror", and add it to this repository.  The current three PRs that are open and perform this mirroring function, I feel should be merged ahead of time since they a) work towards the 2019 Key Result, and b) are blocking changes for our feature sorting script, but after that, perhaps perform said script on folders like API, JavaScript, etc.?

File: css/properties/box-decoration-break.json [hunk 1]
```diff
@@ -73,7 +73,8 @@
               "version_added": true
             },
             "webview_android": {
-              "version_added": true,
+              "prefix": "-webkit-",
+              "version_added": "≤37",
```
Comment (jpmedley) [inline css/properties/box-decoration-break.json]: Look above. `chrome_android` for this feature should be 25. It's listed as 18.

File: css/properties/border-top-right-radius.json [hunk 2]
```diff
@@ -102,9 +102,15 @@
...
-            }
+            "webview_android": [
+              {
+                "version_added": "≤37"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "≤37"
+              }
+            ]
```
Comment (jpmedley) [inline css/properties/border-top-right-radius.json]: `chrome_android` needs some attention for this item.

File: css/properties/border-top-left-radius.json [hunk 3]
```diff
@@ -102,9 +102,15 @@
...
-            }
+            "webview_android": [
+              {
+                "version_added": "≤37"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "≤37"
+              }
+            ]
```
Comment (jpmedley) [inline css/properties/border-top-left-radius.json]: `chrome_android` needs some attention for this one.

File: css/properties/border-bottom-right-radius.json [hunk 4]
```diff
@@ -102,9 +102,15 @@
...
-            }
+            "webview_android": [
+              {
+                "version_added": "≤37"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "≤37"
+              }
+            ]
```
Comment (jpmedley) [inline css/properties/border-bottom-right-radius.json]: `chrome_android` needs some attention.

**Comment by vinyldarkscratch** [comment]
Thanks for reviewing @jpmedley, however since this PR focuses on WebView-related changes, I think we should focus on Chrome Android changes in another PR/issue.  (This is especially since this mirroring script is planned to be the next bulk update.)  How does the WebView stuff look, however?

**Comment by vinyldarkscratch** [comment]
Actually something I just realized is that the WebView mirroring comes from Chrome Android, so some of the Chrome Android stuff may need to be taken care of first.  Does anything stand out as a potential issue with WebView data (i.e. anything that wouldn't be "37 or lower")?

**Comment by Elchi3** [comment]
I've started reviewing this, but `npm run traverse webview_android css` returns 83 results. Is this only a partial mirroring?

**Comment by vinyldarkscratch** [comment]
A few of them were due to the fact that WebView had no statements, but the other part was arrays.  Two things I should probably revise in my fix script!

**Comment by cincodenada** [comment]
I landed here tracking down changes on `:last-child`, and without context this makes for a confusing end-user experience. I actually still don't know what this means - as-is it implies that WebView stopped supporting `:last-child` (for example) after version 37, which seems unlikely, so I expect something else is going on here. Maybe WebView switched to some other backend after version 37?

There's even [one case](https://github.com/vinyldarkscratch/browser-compat-data/blob/614dcb0738d1ab15656a013ce4dee36401e459a7/css/properties/-webkit-mask-attachment.json#L45-L46) where a feature is listed as both added _and_ removed in `≤37`

I would like to propose an improvement to make things clearer, but I can't figure out what is trying to be communicated by these changes in the first place. Can someone who is more familiar with the context here help explain what the intent here is? Sorry if I'm missing something obvious, I'm a web developer who hasn't done much Android work and thus I am not specifically familiar with WebView or the state thereof.

**Comment by ddbeck** [comment]
@cincodenada For more information about this notation, please see the schema documentation on [_Ranged versions_](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#ranged-versions). The summary is that ≤ values express uncertainty about the exact version a feature was added or removed.

In the cases you've called out, a `version_added` value of ≤37 means, "added to this browser in a version less than or equal to 37" (or put in other words, "added in version 37 but possibly earlier"). A `version_added` value of ≤37 means, "removed in in a version less than or equal to 37" (or put in other words, "removed in version 37 but possibly earlier"). Thus, the pair:

```
"version_added": "≤37",
"version_removed": "≤37"
```

means, roughly, "supported for some releases between the first release and version 37, but the starting and ending values are unknown to us."

If you know with more certainty about when a particular feature was supported or removed from a browser—if you can help us eliminate this uncertainty—I'd welcome PRs or issues amending that data. Thank you!

**Comment by jpmedley** [comment]
@cincodenada I work on Chrome at Google. There is some weirdness about WebView and it's relation to Chrome and Android. For starters, 37 was when WebView became a Chromium fork, which it was not before. I've seen enough incorrect PRs about those early versions that I encourage you to ping me before starting a PR. My email is in my GitHub profile.

**Comment by cincodenada** [comment]
Okay, the Ranged Versions info is the piece I was missing here - I didn't understand that this was an expression of partial information/ambiguity about precisely what version. That makes sense and I see how it's useful to have more information.

Thanks for the lucid examples, @ddbeck, those are very helpful and they all make good sense to me now. I unfortunately don't have more details to add. @jpmedley thanks for the extra info on WebView details, that helps fill in the context as well.

With my better understanding, the browser-compat data itself makes plenty of sense, and is the best it can be given the information we have! I think the tooltip on MDN's docs in this situation could be clearer, and that's a different project - from some searching it looks like that's over at https://github.com/mdn/yari, so I'll see if it makes sense to open an PR over there.

I appreciate the quick and informative responses, and y'all taking the time to clarify the context here even though my question was tangential to this specific issue. Thanks for helping make the web a better place :slightly_smiling_face:

**Pull request merged**